package.json

```diff
@@ -49,6 +49,7 @@
     "js-md5": "0.6.1",
     "jwt-decode": "2.2.0",
     "lib-jitsi-meet": "github:jitsi/lib-jitsi-meet#2be752fc88ff71e454c6b9178b21a33b59c53f41",
+    "libflac": "git+https://github.com/ztl8702/libflac.git#31368097eaf9dcb5ef59365ef60b259cb7b97f07",
```

Comment:
@saghul noticed that your libflac repo is not a clean fork of upstream. Can you please re-created it so the connection is not lost?

Reply:
Ok. (I forget why I decided to copy the repo instead of forking)

Another issue is a bit more complicated, as I manually modified a string in the minified js to let libflac to load its `.mem` file from the `libs` folder. I will need to check if I can do that in the Makefile so that this can be reproducible.

Reply:
Generally, we don't want to use custom forks unless you open a PR upstream. We don't want to become maintainers of this repository.

Reply:
@saghul I will try to raise a PR upstream. It appears now the only modification strictly needed is to add a `package.json` so that the repo can be pulled by npm.

Alternatively, maybe using a post-install script to download the files from github is acceptable?

Reply:
Temporarily using a fork (a proper fork, that is) is acceptable, so long as there a PR open upstream.